### PR TITLE
Speed up feed source retrieval

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Companion front end PR for https://github.com/ibi-group/datatools-server/pull/529. This PR makes use of the new FeedSourceSummary endpoint on the front end to speed up the `FeedSourceTable`.

### CODE REVIEW

**Comment by philip-cline** [comment]
e2e tests, of course, will fail until the back end PR is merged

**Comment by philip-cline** [comment]
Backend is merged, tests should be passing!

**Review by miles-grant-ibigroup — APPROVED** [review]
Have played with this and no regressions! It's a bit faster too

File: lib/types/index.js [hunk 1]
```diff
@@ -431,6 +421,21 @@ export type Feed = {
   versionCount: number
 }
 
+export type FeedSourceSummary = {
```
Comment (binh-dam-ibigroup) [inline lib/types/index.js]: Maybe move the declaration for `FeedSourceSummary` to above the one for `FeedSource` and that could take care of the `name` duplication issue?

**Review by binh-dam-ibigroup — APPROVED** [review]
Feels snappier!

File: lib/types/index.js (hunk 1 above)
Comment (philip-cline) [reply lib/types/index.js]: Sure, that's updated in https://github.com/ibi-group/datatools-ui/pull/981/commits/f23755a3247b1cf848624d1298ce34adffd9717e

**Pull request merged**